packages/spruce-skill-server/tests/mocks/SandboxMock.js

```diff
@@ -33,10 +33,27 @@ module.exports = class SandboxMock {
 			numGuests: options.numGuests || 4
 		})
 
+		const sandboxOther = await this.createSandbox({
```

Comment:
It looks like `createSandbox()` is doing:
`this.skill = await this.createSkill()` on line 79

We should probably pull that out of that method or adjust it so we don't have an orphan `this.skill` that gets overwritten on the 2nd call.  Maybe it becomes `this.otherSkill`?

Also, in `SpruceTest.before()` let's set the other org/location for convenience...i.e.
```
this.otherOrganization = this.mocks.sandbox.otherOrganization
const otherLocationId = Object.keys(this.mocks.sandbox.otherLocations)[0]
this.otherLocation = this.mocks.sandbox.otherLocations[otherLocationId]
this.otherSkill = this.mocks.sandbox.otherSkill
```

Reply:
@kengoldfarb thinking we could move the skill to `sandbox.skill = await this.createSkill()` so the sandbox has its own skills as well? then we can grab different skills if we needed from the different sandboxes.